src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
```

Comment:
Write Interfaces, describe values... for functions. Change in all files.
Hint to discover: [https://mate-academy.github.io/fe-program/js/extra/react-typescript#fetching-data](url)